test/fixtures/worker-runner/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "worker-runner",
+  "version": "1.0.0",
```

Comment:
we can keep this file minimal as https://github.com/holepunchto/pear/blob/main/test/fixtures/versions/package.json#L1-L8